cupy/manipulation/rearrange.py

```diff
@@ -1,6 +1,33 @@
 import cupy
 
 
+def flip(a, axis):
+    """Reverse the order of elements in an array along the given axis.
+
+    Args:
+        a (~cupy.ndarray): Input array.
+        axis (int): Axis in array, which entries are reversed.
+
+    Returns:
+        ~cupy.ndarray: Output array.
+
+    .. seealso:: :func:`numpy.flip`
+
+    """
```

Comment:
It would be also good to explicitly describe that this function was introduced since 1.12.